packages/lab/src/__tests__/__e2e__/banner/Banner.cy.tsx

```diff
@@ -78,3 +45,39 @@ describe("GIVEN a Banner", () => {
     });
   });
 });
+
+describe("WHEN adding BannerCloseButton", () => {
+  beforeEach(() => {
```

Comment:
I would prefer to have a cy.mount per test

Reply:
any particular reason ?

Reply:
It makes changing tests much easier and reduces having to check further up the file to find the mounted component. I think there are a lot of tests here that could be combined too.